### PR TITLE
WIP Compine all

### PR DESCRIPTION
**Pull request by drjova**

### CODE REVIEW

File: backoffice/.envs/local/.django [hunk 1]
```diff
@@ -19,5 +20,5 @@ OPENSEARCH_HOST=opensearch:9200
 OPENSEARCH_INDEX_PREFIX=backoffice-backend-local
 
 # Airflow
-AIRFLOW_BASE_URL=http://host.docker.internal:8082
-AIRFLOW_TOKEN=CHANGE_ME
+AIRFLOW_BASE_URL=http://host.docker.internal:8080
+AIRFLOW_TOKEN=YWlyZmxvdzphaXJmbG93Cg
```
Comment (DonHaul) [inline backoffice/.envs/local/.django]: is it okay to have tokens explicitly defined?

Comment (DonHaul) [reply backoffice/.envs/local/.django]: Also is this token correct? it seems to have an addition `Cg` in the end if we are simply encoding `user:password`

File: backoffice/config/api_router.py [hunk 2]
```diff
@@ -17,9 +17,13 @@
 router.register("users", UserViewSet)
 
 # Workflows
-router.register("workflows/authors", AuthorWorkflowViewSet, basename="workflows-authors"),
+router.register(
+    "workflows/authors", AuthorWorkflowViewSet, basename="workflows-authors"
```
Comment (DonHaul) [inline backoffice/config/api_router.py]: did `black` find these lines too long? Seems like many files have been simply formatted

File: backoffice/backoffice/users/fixtures/tokens.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+[
```
Comment (DonHaul) [inline backoffice/backoffice/users/fixtures/tokens.json:1]: these fixtures are great additions, that greatly improve quality of live when having to create everything from scratch, thanks!

File: backoffice/pyproject.toml [hunk 4]
```diff
@@ -149,6 +149,7 @@ django-elasticsearch-dsl-drf = {git = "https://github.com/cern-sis/django-elasti
 opensearch-dsl = "^2.1.0"
 opensearch-py = "2.6.0"
 djangorestframework-simplejwt = "^5.3.1"
+django-json-widget = "^2.0.1"
```
Comment (DonHaul) [inline backoffice/pyproject.toml:152]: It seems thattThis APP is not being used

File: docker-compose.yaml [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+#                                as described in https://airflow.apache.org/docs/docker-stack/build.html.
+#                                Default: ''
+#
+# Feel free to modify this file to suit your needs.
+---
+x-airflow-common: &airflow-common
+  # In order to add custom dependencies or upgrade provider packages you can use your extended image.
+  # Comment the image line, place your Dockerfile in the directory where you placed the docker-compose.yaml
+  # and uncomment the "build" line below, Then run `docker-compose build` to build the images.
+  image: apache/airflow:2.8.3-python3.11
```
Comment (DonHaul) [inline docker-compose.yaml:51]: This line should be commented, if we specify a `build` below

**Review by DonHaul — CHANGES_REQUESTED** [review]
A very nice cleanup of file we didnt need  + improving ease of use when setting up the project from scratch

Github workflows still seem to be failing, are they now building both an backoffice image and an airflow image?

File: backoffice/.envs/local/.django (hunk 1 above)
Comment (drjova) [reply backoffice/.envs/local/.django]: These are all local tokens, that we define in the fixtures for easier use, we don't use them anywhere

File: dsdsdss.pre-commit-config.yaml
Comment (DonHaul) [inline dsdsdss.pre-commit-config.yaml:1]: what is the purpose of this file

Comment (drjova) [reply dsdsdss.pre-commit-config.yaml:1]: please ignore for the moment the PR is WIP, I just want us to make sure that the services are running first and then I will fix the rest. It wasn't an easy process 🙃

**Pull request merged**